arch/sim/src/sim/up_timer.c

```diff
@@ -0,0 +1,169 @@
+/****************************************************************************
+ * arch/sim/src/sim/up_timer.c
+ *
+ *   Copyright (C) 2014, 2019 Gregory Nutt. All rights reserved.
```

Comment:
All new file need Apache copyright, you can find the right head from the code base.